### PR TITLE
Adding ARM based Node SDK clients for HDInsight

### PR DESCRIPTION
**Pull request by harshiam-hdinsight**
This is the first time we are adding these clients for ARM. So the package version is 0.1.0

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@harshiam-hdinsight__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (harshiam). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/services/HDInsightJob2/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+This project provides a Node.js package for accessing the Azure HDInsight. Right now it supports:
+- **Node.js version: 0.6.15 or higher**
+
+## Features
+
+
+## How to Install
+
+```bash
+npm install azure-hdinsight-jobs
```
Comment (amarzavery) [inline lib/services/HDInsightJob2/README.md]: is this management plane or data plane?

Comment (amarzavery) [inline lib/services/HDInsightJob2/README.md]: And is this targeting arm or asm?
If arm then can we name this as azure-arm-hdinsight-jobs ?

Comment (harshiam-hdinsight) [inline lib/services/HDInsightJob2/README.md]: This is the data plane. Will add the arm keyword and push the change.

**Comment by harshiam-hdinsight** [comment]
@amarzavery Addressed PR comments. please take a look and merge if things look good.

**Comment by harshiam-hdinsight** [comment]
@amarzavery ping...can you please merge this so I can get the package on my side

**Pull request merged**